### ISSUE TITLE
Stylet.Start doesn't work with PackageReference

### ISSUE THREAD

**Issue by Tyrrrz** [canton7/Stylet#58]
Most likely because running scripts in NuGet packages is deprecated.

**Comment by canton7** [comment]
Indeed, the concept of Stylet.Start can never work with PackageReference. Are you trying Stylet with .NET Core 3?

**Comment by Tyrrrz** [comment]
Nope, just a regular .NET Framework project with package reference. I just set it up manually instead, so it's fine.
Yeah, I guess it appears there's no way to make Stylet.Start work with PackageReference then. I added a mention in the wiki: https://github.com/canton7/Stylet/wiki/Quick-Start.
